wagtailinventory/tests/test_helpers.py

```diff
@@ -12,7 +12,7 @@
 
 
 if wagtail.VERSION < (3, 0):  # pragma: nocover
```

Comment:
Could this be removed and below you can then not need to use the `CORE_BLOCKS` and use the full string.

e.g. `wagtail.blocks.field_block.CharBlock`

Reply:
Hi @nickmoreton ,
Good catch! This has been updated in the latest commit https://github.com/cfpb/wagtail-inventory/pull/66/commits/b2e4161501da12d19d4cafe3bb148b301e31c667
This should also resolve the one below.